modules/ROOT/attachments/admin-api.yaml

```diff
@@ -5215,7 +5354,125 @@ components:
             $ref: '#/components/schemas/role_description'
           type: array
       type: object
-
+    namespaced_topic:
+      properties:
+        topic:
+          type: string
+          description: Topic name
+        ns:
+          type: string
+          description: "Topic namespace. If not present, assume that the topic is in the `kafka` namespace"
+      type: object
+    outbound_migration:
+      properties: 
+        migration_type:
+          type: string
+          enum: 
+            - outbound
+        topics:
+          type: array
+          items: 
+            $ref: "#/components/schemas/namespaced_topic"
+          description: List of migrated topics
+        consumer_groups:
+          type: array
+          items: 
+            type: string
+          description: List of migrated consumer groups
```

Comment:
both here and in inbound it is ignored when creating and is not displayed when listing

Reply:
Thanks @bashtanov , I'll remove it.